barcodes_generator_abstract/models/barcode_rule.py

```diff
@@ -49,6 +50,14 @@ class BarcodeRule(models.Model):
         "a new record in the mixed model.",
     )
 
+    is_default_rule = fields.Boolean("Is Default Barcode Rule", copy=False)
```

Comment:
```suggestion
    is_default = fields.Boolean("Is Default Barcode Rule", copy=False)
```
suggestion : at rule level, I think 'is default is enough'.

could you also add an "help" param to mention what this checkbox does ?